tethys_portal/settings.py

```diff
@@ -20,28 +20,28 @@
 https://docs.djangoproject.com/en/3.2/ref/settings/
 """
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+# Build paths inside the project like this: BASE_DIR / '...'
 import os
 import sys
 import yaml
 import logging
 import datetime as dt
+from pathlib import Path
 
 from django.contrib.messages import constants as message_constants
-from tethys_apps.utilities import get_tethys_home_dir
+from tethys_apps.utilities import relative_to_tethys_home
 from tethys_cli.gen_commands import generate_secret_key
 
 from bokeh.settings import settings as bokeh_settings, bokehjsdir
 
 log = logging.getLogger(__name__)
 this_module = sys.modules[__name__]
 
-BASE_DIR = os.path.dirname(__file__)
-TETHYS_HOME = get_tethys_home_dir()
+BASE_DIR = Path(__file__).parent
```

Comment:
I'm a big fan of this change. Now could you do it for the rest of the repo? ;)

Reply:
Looks like over 100 changes. I'm happy to do it, but do you think I should do it in a separate PR?

Reply:
I was sort of joking, but hey if you want to do it that would be awesome. Separate PR for sure.